### ISSUE TITLE
A question about wire merging and RF transfer rates (non-issue prob)

### ISSUE THREAD

**Issue by tzpb8** [BluSunrize/ImmersiveEngineering#917]
I have looked around and i'm not sure where to ask questions or talk about IE sorry about this if this is the wrong place and big thanks for addressing the other issue i talked about earlier:  "#908 HV transformers burning out LV wire"

The engineers manual mentions wire merging but my idea of wire merging isn't happening, i want to somehow send 32 768 RF/t through a HV wire onto the charging station for rapid charging but i just get 4096 RF/t no matter what i try with IE tools and blocks and with two entry points into the charger i was expecting to be able to reach at least 65 536 RF/t eventually but with a lot of testing i'm only getting 4096 RF/t which isn't strange because it is connected to a HV connector which can only input 4096 RF/t but how the heck i bypass that?

Here is how i have tried to do it recently: 

![2016-01-23_07 52 25 - kopia](https://cloud.githubusercontent.com/assets/9417797/12528989/23af48a6-c1a9-11e5-9302-57f8e7599bda.png)

At the top the 8x HV capacitors have the power generated from my generators split evenly among them and i have a HV wire relay at the bottom they are all connected to which in theory should have 8x 4096 RF/t (maximum wire transfer rate according to engineer's manual)

But how can wire "merging" happen when the relays themselves only input/output at 4 096 RF/t?
Should i just give up and settle for a max 4 096 RF/t charging speed? Or is my idea of merging just wrong and it actually is only about merging like.. 8x 512 RF/t wires into one 4096 RF/t hv wire? then i'm sorely mistaken xD

Now i also tried using capacitors directly (two of them even) on the charging station and that worked, i didn't measure it tho so i don't know which method of charging is faster.

Charging the IC2 nanosuit can take up to 48,8 seconds (which isn't a lot, later it will become even worse), which is fine for a ordinary use perhaps, but if i want rapid charging i think it should be a possibility! 

TL;DR: Is it possible to charge faster than 4096 RF/t with the charging station? if so: how can it be done?

extra unrelated thoughts: the external heater, i wanted to use it for the ic2 fermenter but it wouldn't accept it, i know they are very different but a work around to make the external heater maybe compatible would be a block to merge the heat from several external heaters so they act in a similar matter to the IC2 electric heat generator? (you can put in copper coils to increase the heat Unit (hU) with 10 per coil) so i'm not sure how much external heater heats up machines or how they work, so maybe i shouldn't talk about this but just an idea i think is worth throwing out there if the design difference is the reason for the incompatability (one external heater per coil could be directly compatible tho) not even sure if hU is a forge term or just ic2 even, sorry. But this could perhaps be the gate way to more heating stuff in IE as well, not just compatibility with IC2 (i read somewhere that the ic2 crew isn't actually the most liked one so it is understandable if compatibility with ic2 isn't top issue lol, it must be annoying to hear about this.

**Comment by Claycorp** [comment]
Relays are distribution points and thus don't transmit power to what they are connected to and also can carry the 32K RF from relay to relay not relay to connector. HV connectors can only pull out 4K RF from the 32K total of the relay. 

Simple version: Relays move 32K between relays only and don't power anything. Connectors can pull 4K from the relay and power things.

**Comment by tzpb8** [comment]
Hmmm i must have come to the wrong conclusion about the relays, i need to test if wire unification actually is a thing first.

8 connectors onto a single relay that outputs to 8 other connectors, if this works they should transfer at the same rate and the relay shouldn't be a choke point :)

**Comment by Claycorp** [comment]
The choke point is the HV connector. It can only pull out the 4K of the possible 32K total on a HV wire.

**Comment by tzpb8** [comment]
Gotya! i'll take your word for it, too much testing, need more playing :smile:

**Comment by BluSunrize** [comment]
Wire merging means being able to combine 3 HV lines into one for long distance and then splitting them again at the destination. In and output values are lower than wire transfer rates for that exact reason.
The whole point of that system rewrite was to bring a balance into IE's wires, a progression by increasing loss when using HV wires for simple stuff like windmills, and encouraging the user to space his base out a little more by giving them the option to transfer power over distance without a single line for every connection.
HV outputs will no go above 4k within core IE. It makes fora nice hex value and it's more than enough for IE based architecture.
And yes, I know that "it's not enough for big reactors", but: I hate BR, so I don't give a damn.
